.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "editor.formatOnSave": true,
```

Comment:
After looking at all of these configurations ... I would suggest removing **these** `settings.json` as it is **very** personalized

I would like `formatOnSave` in javascript in my VSCode within this project, if you don't want it, use your own configuration
My VSCode also respects prettier

Reply:
Let's then put these settings aside, probably in the new documentation I'm writing.

Reply:
for now, I would say yes